composer.json

```diff
@@ -44,8 +44,9 @@
 		"phpunit/phpunit": "^9.5",
 		"simpod/php-coveralls-mirror": "^3.0",
 		"slevomat/coding-standard": "^8.9",
-		"szepeviktor/phpstan-wordpress": "~1.2.0",
-		"wp-graphql/wp-graphql-testcase": "~2.3"
+		"szepeviktor/phpstan-wordpress": "~1.3.0",
+		"wp-graphql/wp-graphql-testcase": "~2.3",
+		"wp-cli/wp-cli": "2.7.1"
```

Comment:
@jasonbahl v2.8.1 is out so its safe to remove `wp-cli/wp-cli` again

Reply:
Thanks! will update